OpenRA.Mods.Common/Traits/Attack/AttackBase.cs

```diff
@@ -35,10 +35,13 @@ public abstract class AttackBaseInfo : UpgradableTraitInfo
 
 		[VoiceReference] public readonly string Voice = "Action";
 
+		[Desc("Unable to fire while in motion.")]
+		public readonly bool StopToShoot = false;
```

Comment:
The name implies that it should actively stop before shooting, but this just prevents it from firing while moving.  This introduces a situation where the unit may not fire at all, which isn't really what #2327 wanted.